server/datastore/mysql/migrations/tables/20221216160025_AddStreamedColumnToActivities.go

```diff
@@ -17,7 +17,7 @@ func Up_20221216160025(tx *sql.Tx) error {
 		return errors.Wrap(err, "adding streamed column to activities")
 	}
 	if _, err := tx.Exec(
-		"CREATE INDEX activities_streamed_idx ON activities (id, streamed);",
```

Comment:
Why this change? You are ordering by id and then streamed, this should work... does it not?

Reply:
It didn't seem to work:

With `CREATE INDEX activities_streamed_idx ON activities (streamed);`:
```
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: a
   partitions: NULL
         type: ref
possible_keys: activities_streamed_idx
          key: activities_streamed_idx
      key_len: 1
          ref: const
         rows: 1
     filtered: 100.00
        Extra: NULL
*************************** 2. row ***************************
           id: 1
  select_type: SIMPLE
        table: u
   partitions: NULL
         type: eq_ref
possible_keys: PRIMARY
          key: PRIMARY
      key_len: 4
          ref: fleet.a.user_id
         rows: 1
     filtered: 100.00
        Extra: NULL
2 rows in set, 1 warning (0.00 sec)
```

With # With `CREATE INDEX activities_streamed_idx ON activities (id, streamed);`
```
EXPLAIN SELECT a.id, a.user_id, a.created_at, a.activity_type, a.details, coalesce(u.name, a.user_name) as name, u.gravatar_url, u.email, a.streamed FROM activities a LEFT JOIN users u ON _id=u.id) WHERE streamed = 1 \G
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: a
   partitions: NULL
         type: ALL
possible_keys: NULL
          key: NULL
      key_len: NULL
          ref: NULL
         rows: 2
     filtered: 50.00
        Extra: Using where
*************************** 2. row ***************************
           id: 1
  select_type: SIMPLE
        table: u
   partitions: NULL
         type: eq_ref
possible_keys: PRIMARY
          key: PRIMARY
      key_len: 4
          ref: fleet.a.user_id
         rows: 1
     filtered: 100.00
        Extra: NULL
2 rows in set, 1 warning (0.01 sec)
```

Reply:
Oh, because you're not ordering by id, my bad. I think you would've needed to do the index the other way around: streamed, id.